code/modules/mob/living/carbon/xenomorph/damage_procs.dm

```diff
@@ -150,7 +150,7 @@
 /mob/living/carbon/Xenomorph/Ravager/process_rage_damage(damage)
 	if(damage < 1 || world.time < last_damage)
 		return damage
-	rage += round(damage * 0.3) //Gain Rage stacks equal to 30% of damage received.
+	rage += round(damage * 0.25) //Gain Rage stacks equal to 25% of damage received.
```

Comment:
move this value into either a config value, define or caste var and put the comments there.